libs/blocks/chart/chart.js

```diff
@@ -147,32 +164,40 @@ const handleIntersect = (chart, chartOptions) => (entries, observer) => {
   });
 };
 
+const getColors = (authoredColor) => {
+  const colorList = Object.values(colorPalette);
+  if (isNullish(authoredColor) || !colorPalette.hasOwnProperty(authoredColor)) return colorList;
+  const colorIndex = colorList.indexOf(colorPalette[authoredColor]);
+  return colorList.concat(colorList.splice(0, colorIndex));
+};
+
 const getContainerSize = (chartSize, chartType) => {
-  const containerSizes = {
+  const chartHeights = {
```

Comment:
@JasonHowellSlavin Tried to update the data structure based on our previous conversation. Hopefully this reflects what you had described to me as well. Thank you

Reply:
Looks good. Did this require changes anywhere else?

Reply:
No changes needed elsewhere. The only thing was now that I'm grabbing an object that only has height data, I just needed to add the width property to it before returning the object.